src/schema/objects/metadata.yaml

```
@@ -3277,6 +3277,9 @@ StartTime:
   description: |
     Start time in seconds in relation to the start of acquisition of the first
     data sample in the corresponding (neural) dataset (negative values are allowed).
+    This data MAY be specified with sub-second precision using the syntax `s[.000000]`,
+    where `s` reflects seconds, and `000000` reflects fractional seconds. The square
+    brackets indicate that supplying fractional seconds is OPTIONAL.
```

This feels a little over-wordy to me. Is the following clear enough?

```suggestion
    where `s` reflects whole seconds, and `.000000` reflects OPTIONAL fractional seconds.
```